management/server/account.go

```diff
@@ -2206,3 +2226,39 @@ func separateGroups(autoGroups []string, allGroups map[string]*nbgroup.Group) ([
 	}
 	return newAutoGroups, jwtAutoGroups
 }
+
```

Comment:
The AM is already more than 2200 lines monster. New features, and logic should move separated structure. If I see well the new functions are deepened from the store only. 
```
type AccountCache struct {
        store Store
  	getAccountRequests  map[string][]*AccountRequest
	mu                  sync.Mutex
	getAccountRequestCh chan *AccountRequest
}
func NewAccountCache(store Store) *AccountCache{
...
}
func (ac *AccountCache) GetAccountWithBackpressure(ctx context.Context, accountID string) (*Account, error) {
...
}
func (am *AccountCache) processGetAccountBatch(ctx context.Context, accountID string) {
...
}
func (am *AccountCache) processGetAccountRequests(ctx context.Context) {
...
}
```